setup.py

```diff
@@ -35,8 +35,8 @@
     'check-manifest>=0.25',
     'coverage>=4.0',
     'isort>=4.2.2',
-    'jsonresolver[jsonschema]>=0.2.0',
-    'pydocstyle>=1.0.0',
+    'jsonresolver[jsonschema]>=0.2.1',
+    'pydocstyle>=2.0.0',
```

Comment:
We had issues with pydocstyle in other modules (see https://github.com/inveniosoftware/troubleshooting/issues/9). Having the last version will make the future failure more consistent between 'lowest' and 'release' builds.